pkg/sentry/socket/netstack/netstack.go

```diff
@@ -3111,6 +3112,12 @@ func interfaceIoctl(ctx context.Context, io usermem.IO, arg int, ifr *linux.IFRe
 		// Gets the MTU of the device.
 		hostarch.ByteOrder.PutUint32(ifr.Data[:4], iface.MTU)
 
+	case linux.SIOCSIFMTU:
+		mtu := hostarch.ByteOrder.Uint32(ifr.Data[:4])
+		if mtu != iface.MTU {
+			return syserr.ErrNotSupported
+		}
```

Comment:
Add a TODO here point to this issue to actually support setting the MTU.

Reply:
// TODO(gvisor.dev/issue/6033)

Reply:
done :)

Reply:
Minor nit. Normally the TODO will be followed by a description of what remains to be done. eg.

TODO(gvisor.dev/issue/6033): Support setting MTU value.

Reply:
Fixed.